node-uniffi/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "lumina-node-uniffi"
+version = "0.1.0"
+edition = "2021"
+description = "Mobile bindings for Lumina node"
```

```suggestion
description = "Mobile bindings for Lumina node"
publish = false
```
Could you disable publishing for now? I'm not sure if there is a usecase for the uniffi crate being on crates.io, and the `release-plz` tool we are using would publish it right away if merged